README.rst

```diff
@@ -181,7 +180,8 @@ Use additional variables from file while expanding:
 
 The env file should contain `VAR=value` lines. Lines starting with # are ignored.
 
-You can also aggregate and automatically push the result to the target:
+You can also aggregate and automatically push the result to the target, if the
+``target`` option is configured:
```

Comment:
Would it then make sense to also add a documention chunk explaining what happens when target is *not* configured ?

Reply:
Yes, it seems important to talk about the `_git_aggregated` default branch in the documentation.